### DISCUSSION TITLE
How to get user object from the token in api services.

### DISCUSSION THREAD

**Discussion by ovibutt** [brocoders/nestjs-boilerplate#1824] [General]
Hi I am trying to access userid in the services but instead of sending it in body I need to get it from the jwt in the services so that I can add data and add a reference of userid in that row. 
How can we access the user object in service or if this is not possible how can I do it?

**Comment by Shchepotin** [comment]
@ovibutt  Please see the example here https://github.com/brocoders/nestjs-boilerplate/blob/79841b80b7f47209404f043ad251e4865bde00c6/src/auth/auth.controller.ts#L98

**Reply by ovibutt** [reply]
Thank you so much. This solves my issue.
